Week3/homework/step2-6.js

```diff
@@ -4,13 +4,11 @@ const arr2d = [[1, 2], [3, 4], [5, 6]];
 const arr3d = [[[1, 2], [3, 4]], [[5, 6], [7, 8]]];
 
 function flattenArray2d(arr) {
-  // Replace this comment and the next line with your code
-  console.log(arr);
+  return arr.flat(1);
 }
 
 function flattenArray3d(arr) {
-  // Replace this comment and the next line with your code
-  console.log(arr);
+  return arr.flat(2);
```

Comment:
👍. But since this method is quite new and wouldn't be supported by all browsers, I would suggest that you read about a topic in programming named as recursion.